### PR TITLE
Style 404 page

### LINKED ISSUE #1378: Page not found page needs padding on left side

### PR DESCRIPTION
**Pull request by jfrankl**
Fixes #1378.

image

### CODE REVIEW

File: src/nyc_trees/apps/core/templates/404.html [hunk 1]
```diff
@@ -1,6 +1,9 @@
-{% extends "base.html" %}
+{% extends "registration/registration_base.html" %}
+{% load i18n %}
 
-{% block content %}
-    <h1>Page not found</h1>
-    <p>Sorry, but the requested page could not be found.</p>
+{% block registration_title %}{% trans "Page Not Found" %}{% endblock %}
```
Comment (steventlamb) [inline src/nyc_trees/apps/core/templates/404.html]: I thought translation wasn't on the table for this app, curious about the choice to use it.

File: src/nyc_trees/apps/core/templates/404.html [hunk 2]
```diff
@@ -1,6 +1,9 @@
...
+
+{% block registration_content %}
+  <h4>This is 404 error</h4>
```
Comment (steventlamb) [inline src/nyc_trees/apps/core/templates/404.html]: I'm not sure this is grammatically correct.

Comment (kdeloach) [inline src/nyc_trees/apps/core/templates/404.html]: image

Comment (steventlamb) [inline src/nyc_trees/apps/core/templates/404.html]: Got it, nevermind.

**Comment by steventlamb** [comment]
tested, works as advertised. +1, barring comments.

**Pull request merged**